packages/twenty-front/jest.config.ts

```diff
@@ -19,7 +21,6 @@ export default {
       functions: 60,
     },
   },
-  collectCoverage: true,
```

Comment:
Updated this to be consistent across the board, coverage should be disabled by default.